src/ResourceManager/ApiManagement/Commands.ApiManagement/Commands/NewAzureApiManagementVirtualNetwork.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Azure.Commands.ApiManagement.Commands
     using System;
     using System.Management.Automation;
 
-    [Cmdlet(VerbsCommon.New, "AzureRmApiManagementVirtualNetwork"), OutputType(typeof(PsApiManagementVirtualNetwork))]
+    [Cmdlet(VerbsCommon.New, "AzureRmApiManagementVirtualNetwork", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.None), OutputType(typeof(PsApiManagementVirtualNetwork))]
```

Comment:
You will need to implement should process in cdmlet execution.  See: https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d for details